src/Components/DurationLineChart.tsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react'
+import { Attempt } from '../types'
+
+import {
+  VictoryAxis,
+  VictoryChart,
+  VictoryLabel,
+  VictoryLine,
+  VictoryScatter,
+  VictoryTheme,
+  VictoryTooltip,
+  VictoryVoronoiContainer
+} from 'victory'
+
+//component for rendering the vertical line on the line chart
+class Cursor extends React.Component<any> {
```

Comment:
Why not define your prop types instead of using any?

Reply:
Using any here because I can literally not figure out how to make this compile and run without it.  I can't figure out the type of scale, or how to pass x or scale as props from inside the VictoryVoronoiContainer component.  I'm sure there's a way but right now I've wasted enough time trying to get it to work that I'm moving forward with this.  I'll leave a todo to come back and try to fix this later though